packages/driver/src/cy/commands/waiting.js

```diff
@@ -138,7 +138,9 @@ module.exports = (Commands, Cypress, cy, state) => {
         log.set('referencesAlias', aliases)
       }
 
-      if (command.get('name') !== 'route') {
+      const isRouteCommand = () => command.get('name') === 'route' || command.get('subject').xhrUrl
```

Comment:
@jennifer-shehane I am not sure about this line. Can we assume that if `xhrUrl` is present in `subject` then it is a `route` command?

I mean, this is because I've added `cy.fixture` which wraps the `route`. So, the name of the final command is `then`... parent is `fixuture` and second parent is `route`. But subject stays the same. I am just wondering if it could stay as it is or should I be more explicite and check parents:

```
const isRouteCommandWithFixtureCheck = () => {
   if (command.get('name') === 'then') {
       const parent =command.get('prev');
       const root = parent.get('prev')
       return parent.get('name') === 'fixture' && root.get('name') === 'route'
    }
}

if (command.get('name') !== 'route' && !isRouteCommandWithFixtureCheck()) {
   // raise the error
}
```

Reply:
I am not sure, @jennifer-shehane do you know by any chance this?

Reply:
Ok, with `cy.now` this is not longer a problem :)